Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -144,6 +144,9 @@ template<> inline LineClampValue fromCSSValue(const CSSValue& value)
 {
     auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
 
+    if (primitiveValue.isValueID() && primitiveValue.valueID() == CSSValueNone)
```

Comment:
I believe there is no need to check is `isValueID` because `valueID` returns `CSSValueInvalid` in that case. I think we did that to make code like this easier to write.

Reply:
Also, another way to do this is to assert the `valueID` is `CSSValueNone` instead of `ASSERT_NOT_REACHED` and not any new code.